benchmarks/benchmarks/stats.py

```diff
@@ -249,6 +249,84 @@ def time_distribution(self, dist_name, method):
         self.method(*self.args, **self.kwds)
 
 
+class TrackContinuousRoundtrip(Benchmark):
+    # Benchmarks that track a value(s) for every distribution can go here
```

Comment:
```suggestion
    # Benchmarks that track a value for every distribution can go here
```

Some of these do test the maximum error over a variety of cases, but either way each benchmark only reports 1 value.